reference/api/pubsub_api.md

```diff
@@ -61,6 +61,7 @@ Example:
 ```json
 [
   {
+    "pubsubName": "pubsub",
```

Comment:
Apologies, I made a mistake (happens.. every now and then :)). It's in fact `pubsubname`.